code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1521,13 +1521,25 @@
 /obj/item/weapon/reagent_containers/food/snacks/no_raisin
 	name = "4no raisins"
 	icon_state = "4no_raisins"
-	desc = "Best raisins in the universe. Not sure why."
+	desc = "Best raisins in the universe. They've been FORtified with a number (no.) of nutrients, hence the name."
```

Comment:
It is illegal to lie in marketing.

Reply:
> 
> 
> These are actually better than 4noraisins by the dollar. You can buy out 11 of them for $33 and get more nutrition than a single 4noraisins at $35. You'd just need to find a second Dan machine.

Ah yes, I'll gladly wait the thousand seconds per sale to mass buy the cheap raisins.

Reply:
To make that line accurate, it would need 4 nutriment, not 6, forever ruining the raisin meta.

Reply:
Each of the 4 world class nutrients that go into 4noraisins are worth 1.5 nutriment each. That's just how nutritious they are.